metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java

```diff
@@ -108,12 +114,10 @@ private Path extractArchive(Path archiveFile) throws HarvesterException {
     } catch (IOException e) {
       throw new HarvesterException("Problem correcting directory rights.", e);
     }
-
-    // Return the extracted directory
-    return extractedDirectory;
   }
 
-  private Path downloadFile(String archiveUrlString, Path downloadDirectory) throws IOException, URISyntaxException {
+  @Override
+  public Path downloadFile(String archiveUrlString, Path downloadDirectory) throws IOException, URISyntaxException {
```

Comment:
Since this now becomes a public method we'd need to improve it by throwing HarvesterException instead of these two, similarly to the other methods.